### ISSUE TITLE
Error page does not fully work without JS

### ISSUE THREAD

**Issue by psvenk** [qutebrowser/qutebrowser#6204]
When a page cannot load due to an error (e.g., renderer process killed), an error page with the text "Unable to load page" is displayed with the URL of the page that I tried to load and a button to "Try again"; however, clicking on this button has no effect. Further inspection reveals that this button relies on JavaScript (I have `content.javascript.enabled` set to false by default).

As a small quality of life improvement, would it be feasible to change the button to a simple link that would work without JS? I would be happy to open a pull request with this change (it seems that this would only entail changing a few lines in [`error.html`](https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/html/error.html)); I just wanted to check first if that would be a good idea.

**Comment by The-Compiler** [comment]
Hmm, this makes me wonder why this even was using JS in the first place...

When the error pages were added in e56ed54102f4545f88a11d78d9fcb4b252385cab, we called `location.reload()` instead. With 1fcce6d87cdcd11cb9becd2a022fce028110a639 I changed this to use `location.href = url` (with a fixup in 5c367e7ab24bb3b885afe0eb7c1c95ad0caf2634, and some more follow-ups with e81f070a7929e6c7a1f4ff097c7106a5d256d90e and d2e5eae7fb32a8ddd63f67357b972476c399a6a1).

In hindsight, a normal link makes perfect sense, and I don't see why this should be using JS to be honest... So go ahead!
